doc/Command-Reference.md

```diff
@@ -4528,7 +4543,7 @@ This command displays the user persistet-watermark for the queues (Egress shared
   admin@sonic:~$ show queue persistent-watermark multicast
   ```
 
-- NOTE: Both "user watermark" and "persistent watermark" can be cleared by user:
+- NOTE: "user watermark", "persistent watermark" and "ingress dropped packets" can be cleared by user:
```

Comment:
please also include the command to clear the pg drop counters

Reply:
@neethajohn  done